sympy/stats/crv.py

```diff
@@ -78,7 +78,7 @@ def as_boolean(self):
 class ConditionalContinuousDomain(ContinuousDomain, ConditionalDomain):
     """
     A domain with continuous support that has been further restricted by a
-    condition such as x > 3
+    condition such as x > 3.
```

Comment:
```suggestion
    condition such as $x > 3$.
```
I'd try to abbreviate these formulas with LaTeX or inline literals for this or any other similar occurrences of this.
The golden rule I'd use is
1. If the symbols are short (`x, y, x_1, ...`) and have well-defined LaTeX syntax, try to use LaTeX `$ $`
2. If the symbols are long or use everyday words (`foo, bar, foo_bar, ...`), try to use inline literal ` `` `` `
3. If you are not confident with, try to use inline literal ` `` `` ` than experimenting with LaTeX

Reply:
Yes. Will do this.